Doc/library/pdb.rst

```diff
@@ -241,10 +241,11 @@ middle of a quoted string.
    triple: debugger; configuration; file
 
 If a file :file:`.pdbrc` exists in the user's home directory or in the current
-directory, it is read in and executed as if it had been typed at the debugger
-prompt.  This is particularly useful for aliases.  If both files exist, the one
-in the home directory is read first and aliases defined there can be overridden
-by the local file.
+directory, it is read with ``'utf-8'`` encoding rather than system locale
+encoding and executed as if it had been typed at the debugger prompt.This is
```

Comment:
And please minimize the diff to make reviewing easier. Do not rewrap the unchaged text. It is normal if some lines will be too short or slightly longer that PEP 8 recommendations.

Reply:
Current diff is the one the best I can come up. Not sure we would recommend a line with 120 chars?